lib/npm_ignore.test.ts

```diff
@@ -9,45 +9,61 @@ Deno.test("should include src directory when the source files are not necessary"
     sourceMaps: undefined,
     inlineSources: undefined,
     expectHasSrcFolder: true,
+    includeScriptModule: true,
+    includeEsModule: true,
   });
   runTest({
     sourceMaps: true,
     inlineSources: undefined,
     expectHasSrcFolder: false,
+    includeScriptModule: true,
+    includeEsModule: true,
   });
   runTest({
     sourceMaps: "inline",
     inlineSources: undefined,
     expectHasSrcFolder: false,
+    includeScriptModule: true,
+    includeEsModule: true,
   });
 
   runTest({
     sourceMaps: true,
     inlineSources: false,
     expectHasSrcFolder: false,
+    includeScriptModule: true,
+    includeEsModule: true,
   });
 
   runTest({
     sourceMaps: undefined,
     inlineSources: true,
     expectHasSrcFolder: true,
+    includeScriptModule: true,
+    includeEsModule: true,
   });
   runTest({
     sourceMaps: true,
     inlineSources: true,
     expectHasSrcFolder: true,
+    includeScriptModule: true,
+    includeEsModule: true,
   });
   runTest({
     sourceMaps: "inline",
     inlineSources: true,
     expectHasSrcFolder: true,
+    includeScriptModule: true,
+    includeEsModule: true,
```

Comment:
It seems these are always true? Maybe let's add a test for each false and then both true?

Reply:
Do you mean a separate copy of the whole `"should include src directory when the source files are not necessary"` or just modify the values of the current `runTest`s?